app/src/main/java/org/oppia/app/options/OptionsAppLanguageViewModel.kt

```diff
@@ -0,0 +1,6 @@
+package org.oppia.app.options
+
+/** App language settings view model for the recycler view in [OptionsFragment]. */
+class OptionsAppLanguageViewModel : OptionsItemViewModel() {
+  var appLanguage: String = ""
```

Comment:
Instead of var make it val `ObservableField`
Ditto elsewhere.

```
/** App language settings view model for the recycler view in [OptionsFragment]. */
class OptionsAppLanguageViewModel : OptionsItemViewModel() {
  val appLanguage = ObservableField<String>("")

  fun setAppLanguage(appLanguageValue: String) {
    appLanguage.set(appLanguageValue)
  }
}

```

Reply:
Done.